software/spmd/bsg_print_stat/Makefile

```diff
@@ -1,5 +1,3 @@
-bsg_tiles_org_X =0
-bsg_tiles_org_Y =1
```

Comment:
Why are these no longer necessary? Do we set them in the SPMD loader now? (sort of like how we set them in CUDA)

Reply:
the default is defined in mk/Makefile.dimensions. this is passed to nbf.py to create nbf file.

Reply:
This is so that it makes it easier to switch the coordinate for the top-left tile, so we can just change the Makefile.dimension, instead of modifying every Makefile.

Reply:
Makes sense, thanks for explaining!